README.md

```diff
@@ -248,16 +248,24 @@ func application(_ application: UIApplication, continue userActivity: NSUserActi
 	return [IterableAPI handleUniversalLink:userActivity.webpageURL];
 }
 ```
+
+### Deferred deep linking
+
+If an app user does not have your app installed, deep linking will not work. The user will need to install your app first. After installing your app, the user will be taken to a specific location within your app when they launch it for the first time. See definition of deferred deep linking [here](#https://en.wikipedia.org/wiki/Deferred_deep_linking).
```

Comment:
- I'm not quite sure about this part—maybe flesh out the sequence of events a bit more? So, a user clicks on a link that would take you to your app, but you don't have it installed... so it directs you to the app store, installs it, and when you open it, it takes you to the desired content? Is that how it works? I think it's a little confusing to say "deep linking will not work," because with this feature deep linking _will_ work even if you haven't installed the app yet, right?

Reply:
You are right about how ddl works. Let me see if I can rephrase it better.